osgp/protocol-adapter-mqtt/osgp-protocol-simulator-mqtt/README.md

```diff
@@ -4,6 +4,10 @@ Maven build:
 ```
 mvn clean install
 ```
+The build generates both .war file which can be deployed in Tomcat.
```

Comment:
The use of "both" suggests there should be 2 things?
Did you mean both a .war file and a docker image? Or could "both" be dropped here?

Reply:
Dropped 'both'